File: sample_java/src/main/java/com/shong/sample_java/MainActivity2.java
package com.shong.sample_java;

import android.os.Bundle;
import android.view.View;

import androidx.activity.ComponentActivity;
import androidx.appcompat.app.AppCompatActivity;

import com.shong.klog.Klog;

public class MainActivity2 extends AppCompatActivity {
    Klog log = Klog.INSTANCE;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main2);

        findViewById(R.id.goBackButton).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                finish();
            }
        });

        findViewById(R.id.makeLogButton).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                log.f("" + System.currentTimeMillis(), "test log " + System.currentTimeMillis());
            }
        });

        findViewById(R.id.showTestButton).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                log.f("testKey", "show Test!");
            }
        });

        findViewById(R.id.removeKeyButton).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                log.removeFloatLog("testKey");
            }
        });

        findViewById(R.id.removeAllFloatButton).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                log.removeAllFloatLog();
            }
        });

        ComponentActivity activity = this;
        findViewById(R.id.runFloatingButton).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                log.runFloating(activity);
            }
        });

        findViewById(R.id.stopFloatingButton).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                log.stopFloating(activity);
            }
        });
    }
}

File: sample_kotlin/src/main/java/com/shong/sample_kotlin/MainActivity2.kt
package com.shong.sample_kotlin

import android.annotation.SuppressLint
import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity
import com.shong.klog.Klog
import java.text.SimpleDateFormat

class MainActivity2 : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main2)

        findViewById<Button>(R.id.goBackButton).setOnClickListener {
            finish()
        }

        findViewById<Button>(R.id.makeLogButton).setOnClickListener {
            Klog.f("${System.currentTimeMillis()}", "test log ${System.currentTimeMillis()}")
        }

        findViewById<Button>(R.id.showTestButton).setOnClickListener {
            Klog.f("testKey", "show Test!")
        }

        findViewById<Button>(R.id.removeKeyButton).setOnClickListener {
            Klog.removeFloatLog("testKey")
        }

        findViewById<Button>(R.id.removeAllFloatButton).setOnClickListener {
            Klog.removeAllFloatLog()
        }

        findViewById<Button>(R.id.runFloatingButton).setOnClickListener {
            Klog.runFloating(this)
        }

        findViewById<Button>(R.id.stopFloatingButton).setOnClickListener {
            Klog.stopFloating(this)
        }
    }
}